[Flaky Test] go_test: //staging/src/k8s.io/apiserver/pkg/server/go_default_test:run_2_of_2

<!-- Please only use this template for submitting reports about flaky tests or jobs (pass or fail with no underlying change in code) in Kubernetes CI -->

**Which jobs are flaking**:
bazel-test-master
**Which test(s) are flaking**:
go_test: //staging/src/k8s.io/apiserver/pkg/server/go_default_test:run_2_of_2
**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#bazel-test-master
**Reason for failure**:
```
exec ${PAGER:-/usr/bin/less} "$0" || exit 1
Executing tests from //staging/src/k8s.io/apiserver/pkg/server:go_default_test
-----------------------------------------------------------------------------
W1211 19:43:03.014383  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.015049  151627 authentication.go:40] Authentication is disabled
W1211 19:43:03.025706  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.026223  151627 authentication.go:40] Authentication is disabled
E1211 19:43:03.028245  151627 pathrecorder.go:107] registered "/healthz" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0x2aa8429)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0x2aa8429, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0x2aa8429, 0x8, 0x2dbd000, 0xc0004b7400)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:165 +0x3a2
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.028433  151627 pathrecorder.go:107] registered "/healthz/ping" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044f880)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044f880, 0xd)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044f880, 0xd, 0x2dbd000, 0xc000496610)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.029786  151627 pathrecorder.go:107] registered "/healthz/log" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044f8c0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044f8c0, 0xc)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044f8c0, 0xc, 0x2dbd000, 0xc000496670)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.031110  151627 pathrecorder.go:107] registered "/healthz/delegate-health" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0002d8ea0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0002d8ea0, 0x18)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0002d8ea0, 0x18, 0x2dbd000, 0xc0004966d0)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.032466  151627 pathrecorder.go:107] registered "/healthz/poststarthook/generic-apiserver-start-informers" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0005c7600)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0005c7600, 0x38)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0005c7600, 0x38, 0x2dbd000, 0xc000496730)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.033795  151627 pathrecorder.go:107] registered "/healthz/poststarthook/max-in-flight-filter" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0004bcf00)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0004bcf00, 0x2b)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0004bcf00, 0x2b, 0x2dbd000, 0xc000496790)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.035167  151627 pathrecorder.go:107] registered "/healthz/poststarthook/delegate-post-start-hook" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0004bcf30)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0004bcf30, 0x2f)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0004bcf30, 0x2f, 0x2dbd000, 0xc0004967f0)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa8429, 0x8, 0xc000432e80, 0x6, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:132
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installHealthz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:96 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:298 +0xbe
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.036550  151627 pathrecorder.go:107] registered "/livez" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0x2aa550a)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0x2aa550a, 0x6)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0x2aa550a, 0x6, 0x2dbd000, 0xc0004b7540)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa550a, 0x6, 0xc000433080, 0x5, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:165 +0x3a2
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallLivezHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:148
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installLivez(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:112 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:299 +0xcf
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.037846  151627 pathrecorder.go:107] registered "/livez/ping" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044f930)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044f930, 0xb)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044f930, 0xb, 0x2dbd000, 0xc000496900)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa550a, 0x6, 0xc000433080, 0x5, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallLivezHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:148
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installLivez(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:112 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:299 +0xcf
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.038994  151627 pathrecorder.go:107] registered "/livez/log" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044f9a0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044f9a0, 0xa)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044f9a0, 0xa, 0x2dbd000, 0xc000496960)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa550a, 0x6, 0xc000433080, 0x5, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallLivezHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:148
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installLivez(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:112 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:299 +0xcf
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.040108  151627 pathrecorder.go:107] registered "/livez/poststarthook/generic-apiserver-start-informers" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0005c76c0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0005c76c0, 0x36)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0005c76c0, 0x36, 0x2dbd000, 0xc0004969c0)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa550a, 0x6, 0xc000433080, 0x5, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallLivezHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:148
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installLivez(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:112 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:299 +0xcf
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.041214  151627 pathrecorder.go:107] registered "/livez/poststarthook/max-in-flight-filter" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0004bcff0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0004bcff0, 0x29)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0004bcff0, 0x29, 0x2dbd000, 0xc000496a20)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa550a, 0x6, 0xc000433080, 0x5, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallLivezHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:148
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installLivez(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:112 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:299 +0xcf
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.042329  151627 pathrecorder.go:107] registered "/livez/poststarthook/delegate-post-start-hook" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0004bd020)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0004bd020, 0x2d)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0004bd020, 0x2d, 0x2dbd000, 0xc000496a80)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa550a, 0x6, 0xc000433080, 0x5, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallLivezHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:148
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installLivez(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:112 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:299 +0xcf
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.043396  151627 genericapiserver.go:302] Failed to install readyz shutdown check unable to add because the readyz endpoint has already been created
E1211 19:43:03.043629  151627 pathrecorder.go:107] registered "/readyz" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0x2aa6b6e)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0x2aa6b6e, 0x7)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0x2aa6b6e, 0x7, 0x2dbd000, 0xc0004b7680)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:165 +0x3a2
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.044699  151627 pathrecorder.go:107] registered "/readyz/ping" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044faa0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044faa0, 0xc)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044faa0, 0xc, 0x2dbd000, 0xc000496bb0)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.045901  151627 pathrecorder.go:107] registered "/readyz/log" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044fad0)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044fad0, 0xb)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044fad0, 0xb, 0x2dbd000, 0xc000496c10)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.047054  151627 pathrecorder.go:107] registered "/readyz/poststarthook/generic-apiserver-start-informers" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0005c7780)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0005c7780, 0x37)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0005c7780, 0x37, 0x2dbd000, 0xc000496c70)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.048152  151627 pathrecorder.go:107] registered "/readyz/informer-sync" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0002d8f20)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0002d8f20, 0x15)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0002d8f20, 0x15, 0x2dbd000, 0xc000496cd0)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.049289  151627 pathrecorder.go:107] registered "/readyz/poststarthook/max-in-flight-filter" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0004bd110)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0004bd110, 0x2a)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0004bd110, 0x2a, 0x2dbd000, 0xc000496d30)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.050402  151627 pathrecorder.go:107] registered "/readyz/poststarthook/delegate-post-start-hook" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc0004bd170)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc0004bd170, 0x2e)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc0004bd170, 0x2e, 0x2dbd000, 0xc000496d90)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
E1211 19:43:03.051539  151627 pathrecorder.go:107] registered "/readyz/shutdown" from goroutine 88 [running]:
runtime/debug.Stack(0x3c91e20, 0xc000414f60, 0xc00044fb40)
	GOROOT/src/runtime/debug/stack.go:24 +0xab
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).trackCallers(0xc0005fe0e0, 0xc00044fb40, 0x10)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:109 +0xe5
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/mux.(*PathRecorderMux).Handle(0xc0005fe0e0, 0xc00044fb40, 0x10, 0x2dbd000, 0xc000496df0)
	staging/src/k8s.io/apiserver/pkg/server/mux/pathrecorder.go:173 +0xca
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallPathHandler(0x2dbad80, 0xc0005fe0e0, 0x2aa6b6e, 0x7, 0xc000432f00, 0x7, 0x8)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:177 +0x5da
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server/healthz.InstallReadyzHandler(...)
	staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go:140
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).installReadyz(0xc0007e2800)
	staging/src/k8s.io/apiserver/pkg/server/healthz.go:104 +0x192
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.(*GenericAPIServer).PrepareRun(0xc0007e2800, 0x2abefd8)
	staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go:304 +0x285
k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/server.TestNewWithDelegate(0xc0007ea300)
	staging/src/k8s.io/apiserver/pkg/server/config_test.go:110 +0x7e9
testing.tRunner(0xc0007ea300, 0x2b76308)
	GOROOT/src/testing/testing.go:1123 +0x203
created by testing.(*T).Run
	GOROOT/src/testing/testing.go:1168 +0x5bc
--- FAIL: TestNewWithDelegate (0.08s)
    --- FAIL: TestNewWithDelegate/http://127.0.0.1:34265/healthz (0.00s)
        config_test.go:210: HTTP/1.1 500 Internal Server Error
            Content-Length: 334
            Cache-Control: no-cache, private
            Content-Type: text/plain; charset=utf-8
            Date: Fri, 11 Dec 2020 19:43:03 GMT
            X-Content-Type-Options: nosniff
            
            [+]ping ok
            [+]log ok
            [-]wrapping-health failed: reason withheld
            [-]delegate-health failed: reason withheld
            [+]poststarthook/generic-apiserver-start-informers ok
            [-]poststarthook/max-in-flight-filter failed: reason withheld
            [+]poststarthook/delegate-post-start-hook ok
            [+]poststarthook/wrapping-post-start-hook ok
            healthz check failed
            
        config_test.go:218: "http://127.0.0.1:34265/healthz" expected [+]ping ok
            [+]log ok
            [-]wrapping-health failed: reason withheld
            [-]delegate-health failed: reason withheld
            [+]poststarthook/generic-apiserver-start-informers ok
            [+]poststarthook/max-in-flight-filter ok
            [+]poststarthook/delegate-post-start-hook ok
            [+]poststarthook/wrapping-post-start-hook ok
            healthz check failed
            , got [+]ping ok
            [+]log ok
            [-]wrapping-health failed: reason withheld
            [-]delegate-health failed: reason withheld
            [+]poststarthook/generic-apiserver-start-informers ok
            [-]poststarthook/max-in-flight-filter failed: reason withheld
            [+]poststarthook/delegate-post-start-hook ok
            [+]poststarthook/wrapping-post-start-hook ok
            healthz check failed
W1211 19:43:03.091684  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.092944  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.093433  151627 authentication.go:40] Authentication is disabled
W1211 19:43:03.095582  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.095860  151627 authentication.go:40] Authentication is disabled
W1211 19:43:03.131934  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.132633  151627 authentication.go:40] Authentication is disabled
W1211 19:43:03.182529  151627 authorization.go:47] Authorization is disabled
W1211 19:43:03.182880  151627 authentication.go:40] Authentication is disabled
W1211 19:43:03.228301  151627 authentication.go:40] Authentication is disabled
I1211 19:43:03.237855  151627 secure_serving.go:241] Stopped listening on [::]:41573
FAIL
```
**Anything else we need to know**:
- links to go.k8s.io/triage appreciated
  [Will update shortly]
- links to specific failures in spyglass appreciated
   * https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/periodic-kubernetes-bazel-test-master/1337477929449295872
<!-- Please see the deflaking doc (https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/flaky-tests.md) for more guidance! -->

/sig api-machinery
@kubernetes/ci-signal

--- a/staging/src/k8s.io/apiserver/pkg/server/config_test.go
+++ b/staging/src/k8s.io/apiserver/pkg/server/config_test.go
@@ -30,6 +30,7 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"k8s.io/apimachinery/pkg/util/json"
 	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apimachinery/pkg/util/waitgroup"
 	auditinternal "k8s.io/apiserver/pkg/apis/audit"
 	"k8s.io/apiserver/pkg/audit"
@@ -179,6 +180,24 @@ func TestNewWithDelegate(t *testing.T) {
 		"/readyz/shutdown",
 	}
 	checkExpectedPathsAtRoot(server.URL, expectedPaths, t)
+
+	// wait for health (max-in-flight-filter is initialized asynchronously, can take a few milliseconds to initialize)
+	if err := wait.PollImmediate(100*time.Millisecond, wait.ForeverTestTimeout, func() (bool, error) {
+		// healthz checks are installed in PrepareRun
+		resp, err := http.Get(server.URL + "/healthz?exclude=wrapping-health&exclude=delegate-health")
+		if err != nil {
+			t.Fatal(err)
+		}
+		data, _ := ioutil.ReadAll(resp.Body)
+		if http.StatusOK != resp.StatusCode {
+			t.Logf("got %d", resp.StatusCode)
+			t.Log(string(data))
+			return false, nil
+		}
+		return true, nil
+	}); err != nil {
+		t.Fatal(err)
+	}
 	checkPath(server.URL+"/healthz", http.StatusInternalServerError, `[+]ping ok
 [+]log ok
 [-]wrapping-health failed: reason withheld

